packages/web-components/fast-components/src/anchored-region/anchored-region.definition.ts

```diff
@@ -43,7 +49,9 @@ export const fastAnchoredRegionDefinition: WebComponentDefinition = {
                 },
                 {
                     name: "horizontal-default-position",
-                    description: "The horizontal-default-position attribute",
+                    title: "Horizontal default position",
+                    description:
+                        "The default horizontal position of the region relative to the anchor element",
```

Comment:
```suggestion
                        "The default horizontal position of the region relative to the configured 'Anchor'",
```

Reply:
I don't think we have a convention for this yet, but I'm attempting to tie it back to the other property/attribute.